lte/gateway/python/magma/pipelined/app/BUILD.bazel

```diff
@@ -204,6 +204,9 @@ py_library(
 py_library(
     name = "restart_mixin",
     srcs = ["restart_mixin.py"],
+    deps = [
```

Comment:
Curious, why was this not needed before? I see that you added the import to `restart_mixin`, but the `_startup_flow_controller` was there before, right?

Reply:
The import `from magma.pipelined.app.startup_flows import StartupFlows` was newly added

Reply:
> I see that you added the import to `restart_mixin`

My question was, why the import was not needed before.

Reply:
To me it looks like both lines are new?

Reply:
Current master does this: ![Screenshot from 2022-06-10 13-05-01](https://user-images.githubusercontent.com/34488763/173052057-4f328b27-b21e-4677-96e0-dce1135641b4.png)

Reply:
> but the `_startup_flow_controller` was there before

yes, but the object that contains the `StartupFlows` was existing before

Reply:
Yes but it wasn't typed here before so the Python code didn't know what it was in the abstract superclass.